init.lua

```diff
@@ -72,7 +68,9 @@ function toolranks.new_afteruse(itemstack, user, node, digparams)
   if lastlevel < level then
     local levelup_text = "Your " .. toolranks.colors.green ..
                          itemdesc .. toolranks.colors.white ..
-                         " just leveled up!"
+                         " just leveled up to " ..
+                         toolranks.colors.green .. "level " ..
```

Comment:
Making it white is ok for me. It would also be ok for me to leave the "level" word away, as it's redundant anyway. I'm just not a native English speaker, so I'm not sure if "leveled up to 3!" sounds strange. Which would you prefer?